numpy/__init__.py

```diff
@@ -112,6 +112,9 @@
     ComplexWarning, ModuleDeprecationWarning, VisibleDeprecationWarning,
     TooHardError, AxisError)
 
+# Allow distributors to run custom init code before importing numpy.core
+from . import _distributor_init
```

Comment:
This changes this import to happen at build-time rather than only at runtime. Distributors may have content here that relies on numpy being built (a use case for this file is modifying numpy defaults). So I think this should stay inside the `else:` clause. If there's a reason to move it, can you move it to right below `else:`?

Reply:
The problem stems from `from numpy.__config__ import show` since on the meson build that newly imports `_multiarray_umath`, which requires the dll loading from `_distributor_init`. I can move the lines into the `else` clause.